Description: Add missing file contain translation to the POTFILES.in file
 Put string "uglib/pwmd.c" in po/POTFILES.in
 
Forwarded: not-needed
Author: Elías Alejandro Año Mendoza <ealmdz@gmail.com>
Last-Update: 2014-01-19

--- uget.orig/po/POTFILES.in	2014-01-25 21:07:37.000000000 -0500
+++ uget/po/POTFILES.in	2014-01-25 21:16:20.000000000 -0500
@@ -16,6 +16,7 @@
 uglib/UgStdio.c
 uglib/UgUri.c
 uglib/UgUtils.c
+uglib/pwmd.c
 uget-gtk/UgApp-gtk.c
 uget-gtk/UgApp-gtk-aria2.c
 uget-gtk/UgApp-gtk-callback.c
